package.json

```diff
@@ -208,16 +221,35 @@
   "babel": {
     "presets": [
       "./scripts/env",
-      "@babel/preset-react",
+      "@babel/preset-react"
+    ],
+    "plugins": [
+      "dev-expression",
+      "@babel/plugin-syntax-dynamic-import",
+      "@babel/plugin-syntax-import-meta",
+      "@babel/plugin-proposal-class-properties",
+      "@babel/plugin-proposal-json-strings",
       [
-        "@babel/preset-stage-1",
+        "@babel/plugin-proposal-decorators",
         {
-          "decoratorsLegacy": true
+          "legacy": true
         }
-      ]
-    ],
-    "plugins": [
-      "dev-expression"
+      ],
+      "@babel/plugin-proposal-function-sent",
```

Comment:
The list here seems to contain many draft/straw-man level proposals. Any reasons behind the inclusion?